src/extension.ts

```diff
@@ -58,6 +60,9 @@ export const getProsTerminal = async (
 
   await configurePaths(context);
 
+  console.log(`CURRENT PATH: ${process.env.PATH}`);
```

Comment:
I'm not 100% clear how this logger works, will it capture all `console.log` calls or should this line be something like `oneClickLogger.log("")`?

Reply:
After thinking about it a bit more, it might be ideal to use a single logger and a series of tags/message prefixes to sort by later. If we log all output to a single file but prefix messages with something like "ONE-CLICK: log message" or "SIDEBAR-ACTION: log message", then we'll ensure that we get all of the info we might care about easily when asking a user for logs while still being able to filter through the with grep or similar.

Reply:
That setup looks good to me! I can't think of anything else we'd want as a category now, but we can always add more later if needed.